RELEASE-NOTES.txt

```diff
@@ -6,6 +6,7 @@
 - [*] Order editing: disable editing orders in other currencies, due to risk of showing incorrect information [https://github.com/woocommerce/woocommerce-ios/pull/14345]
 - [Internal] Update Zendesk SDK to 9.0.0 [https://github.com/woocommerce/woocommerce-ios/pull/14360]
 - [*] Payments onboarding: for two onboarding errors that show a wp-admin Payments setup URL in a webview ("Payments plugin not setup" and "Stripe overdue" errors that can occur to both WooPayments and Stripe Extension), the webview is now set to Payments Connect instead of Payments Overview. [https://github.com/woocommerce/woocommerce-ios/pull/14297]
+- [*] Blaze: merchants can now select the text for the CTA on their campaigns [https://github.com/woocommerce/woocommerce-ios/issues/14355]
```

Comment:
We usually link the PR here instead of the issue. Feel free to keep it if the GH issue has better details and testing instructions.